esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1871,6 +1871,8 @@ bool CFileSprayEx::onGetDFUExceptions(IEspContext &context, IEspGetDFUExceptions
 
 static StringBuffer &getDropZoneHost(const char *planeName, IPropertyTree *plane, StringBuffer &host)
 {
+    if (isContainerized() && strsame("localhost", host.str())) //ECLWatch may send "localhost" for dropzones without hosts.
+        host.clear();
```

Comment:
@wangkx - please see https://track.hpccsystems.com/browse/HPCC-29303,
which was reported separately and should be committed as a separate issue (so should remove from here).